zowe/zowe-cli
# Fix 'create like' block size issues

**What It Does**
When using the **'Create.dataSetLike()'** File SDK method, the new data set didn't always have the correct or valid block size. This could cause the data set to be unusable. 

**How to Test**
Create a new data set using the **--like** option in the Zowe CLI or the **'Allocate Like'** menu option in Zowe Explorer VS Code extension.  The new data set should always have a block size that matches the 'Like' data set. 

**Review Checklist**
I certify that I have:
- [X ] tested my changes
- [X ] added/updated automated tests
- [X ] updated the changelog
- [X ] followed the [contribution guidelines](https://github.com/zowe/zowe-cli/blob/master/CONTRIBUTING.md)
